girderformindlogger/utility/jsonld_expander.py

```diff
@@ -816,19 +816,24 @@ def formatLdObject(
                 'http://schema.org/url',
                 obj.get('meta', {}).get('protocol', obj).get('url')
             )
+
+            # get protocol data from url (load from database unless refreshCache is set to True)
```

Comment:
I think it's better don't use ternary operator here
initialize your variable and it to empty dict

like that for example:
```
protocol = {}
if protocolUrl is not None:
      protocol = ProtocolModel().getFromUrl(
                protocolUrl,
                'protocol',
                user,
                thread=False,
                refreshCache=refreshCache
            )[0]
```